seismicpro/gather/utils/stats.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+from numba import njit, prange
+
+
+@njit(nogil=True)
```

Comment:
Otherwise `prange` makes no sense

```suggestion
@njit(nogil=True, parallel=True)
```